packages/graphql-cli-graphback-plugin/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "graphql-cli-graphback-plugin",
+  "version": "0.2.1",
+  "description": "Auto generate resolvers on top of the graphql and database of your choice",
+  "homepage": "https://github.com/ankitjena/graphql-cli-graphback-plugin",
+  "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "typescript": {
+    "definition": "types/index.d.ts"
+  },
+  "author": "wtrocki@redhat.com",
+  "scripts": {
+    "clean": "rimraf coverage dist types",
+    "build": "npm run clean && tsc",
+    "cmd": "node ./dist/index.js",
+    "watch": "tsc -w",
+    "lint": "tslint -t stylish --project \"tsconfig.json\""
+  },
+  "engines": {
+    "node": "12.9.1"
+  },
+  "devDependencies": {
+    "@types/chalk": "0.4.31",
+    "@types/inquirer": "6.5.0",
+    "@types/node": "12.7.3",
+    "@types/node-emoji": "1.8.1",
+    "@types/through": "0.0.29",
+    "@types/yargs": "13.0.2",
+    "ava": "2.3.0",
+    "lint-staged": "9.2.4",
+    "rimraf": "3.0.0",
+    "ts-node": "8.3.0",
+    "tslint": "5.19.0",
+    "tslint-config-prettier": "1.18.0",
+    "tslint-microsoft-contrib": "6.2.0",
+    "tsutils": "3.17.1",
+    "typescript": "3.5.3",
+    "typescript-eslint-parser": "21.0.2"
+  },
+  "dependencies": {
+    "chalk": "2.4.2",
+    "chokidar": "3.0.2",
+    "execa": "2.0.4",
+    "figlet": "1.2.3",
+    "glob": "7.1.4",
+    "graphback-cli": "^0.8.3",
+    "inquirer": "7.0.0",
+    "node-emoji": "1.10.0",
+    "ora": "3.4.0",
+    "parse-github-url": "1.0.2",
+    "pg": "7.12.1",
+    "request": "2.88.0",
+    "sqlite3": "4.1.0",
+    "tar": "4.4.10",
+    "tmp": "0.1.0",
+    "yargs": "14.0.0"
+  },
+  "jest": {
+    "transform": {
+      "^.+\\.tsx?$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+    },
+    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(tsx?|jsx?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "json",
+      "jsx"
+    ]
+  },
+  "license": "ISC"
```

Comment:
We need to relicense everything to MIT

Reply:
The repo has an Apache-2.0 License

Reply:
This raises an important question - Should the license file always be distributed with the source? Because in our monorepo we have the license at a top level but if you actually installed one of these packages from npm, the license would not be in there. WDYT?

Reply:
Graphql-cli is MIT and to integrate it we need to be MIT. Rest of the packages can be in any license.

Reply:
@wtrocki I think we should probably think about how/where the license files are placed then. Most companies/end users simply cannot touch code if there is any ambiguity around licensing. If we're planning to have different packages using different licenses all within the one repo, I think we should probably be very explicit and include the appropriate license file directly in each package folder. Otherwise the released distribution they download from npm will not actually contain a license file. What do you think?